pkg/systemd/services/service-details.jsx

```diff
@@ -217,11 +226,15 @@ export class ServiceDetails extends React.Component {
             note: "",
             error: "",
             unit_properties: {},
+            showDeleteDialog: false,
```

Comment:
This should use the new dialog protocol introduced in PR #17298 -- sorry for the collision, but let's avoid introducing more instances now.

Reply:
Ok, let's do this in a follow up.

Reply:
https://github.com/cockpit-project/cockpit/pull/17300 has landed, I'd say we should use `Dialogs.show` etc right away.